parsec/core/cli/pki.py

```diff
@@ -187,13 +210,16 @@ def _display_pending_enrollment(pending: PkiEnrollmentSubmitterSubmittedCtx):
 
         else:
             assert isinstance(ctx, PkiEnrollmentSubmitterAcceptedStatusCtx)
-            click.echo(
-                "Enrollment has been accepted on " + click.style(ctx.accepted_on, fg="yellow")
-            )
-            ctx = await ctx.finalize()
+            click.echo(_display_accepted_enrollment(ctx))
             if not dry_run:
-                await save_device_with_selected_auth(
-                    config_dir=config.config_dir, device=ctx.new_device
+                if not await aconfirm("Finalize device creation"):
```

Comment:
What's the use of this confirmation ? I don't see a good usecase for it, especially given it seems redundant with the `--dry-run` option

Reply:
Because the user has to check the information contained in the accepter certificate before finalizing the device creation (similar to what happens on the accepter side)